exercises/alphametics/.meta/template.j2

```diff
@@ -0,0 +1,18 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases[0]["cases"] -%}
+    {% set description = case["description"] | to_snake -%}
+    {% set value = case["input"]["puzzle"] -%}
+    {% if value|length > 100 -%}
+    # Reason to skip this test at https://github.com/exercism/python/pull/1358
```

Comment:
There really wasn't much discussion around the "reason" in that PR. How about:
`"""This test may take a long time to run. Please be patient when running it."""`

I would move the comment to make it a [docstring](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) As well.